script/publish_views

```diff
@@ -4,4 +4,8 @@
 
 cd "$(dirname "$0")/.."
 
-exec python3 -m bigquery_etl.view.publish_views "$@"
+./bqetl bootstrap
+./bqetl view publish "$@"
+
+wait
+trap - EXIT
```

Comment:
what is this waiting for, and why is it resetting the EXIT trap when it's never set?

Reply:
Right, it doesn't seem necessary here. I was copying this over from another script: https://github.com/mozilla/bigquery-etl/blob/main/script/update_example_glean_usage